### PR TITLE
Create new token description

### PR DESCRIPTION
**Pull request by AndreyKortosov**
Added description of new tokens: PUSSY, KXOR, KARMA

### CODE REVIEW

File: src/karma.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: Chameleon KARMA | SORA Docs
+head:
+  - - meta
+    - name: description
+      content: Chameleon pool, liquidity providers in the XOR-ETH pool, buy-back and burn mechanizm, distributed to the Demeter farm, deflationary mechanism
```
Comment (WRRicht3r) [inline src/karma.md]: Here you would add a summary of the content of the page, not just the keywords

Comment (WRRicht3r) [inline src/karma.md]:
```suggestion
      content: The KARMA token provides farming rewards to liquidity providers in the XOR-ETH pool on Polkaswap. It is specifically designed to incentivize liquidity providers and support the stability of the Chameleon pools on Polkaswap.
```

File: src/karma.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  - - meta
+    - name: keywords
+      content: token burn, Chameleon, XOR, ETH, Demeter, KXOR
+---
+
+# Chameleon (KARMA)
+
+KARMA is a key token in the SORA ecosystem, specifically designed to incentivize liquidity providers and support the stability of the Chameleon pools on Polkaswap. Here's a detailed look at how KARMA works.
```
Comment (WRRicht3r) [inline src/karma.md]:
```suggestion
KARMA is a key SORA ecosystem token, specifically designed to incentivize liquidity providers and support the stability of the Chameleon pools on Polkaswap. Here's a detailed look at how KARMA works.
```

File: src/karma.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Purpose and Incentives
+
+The KARMA token is used to provide farming rewards to liquidity providers in the XOR-ETH pool on Polkaswap.
+
+By participating in liquidity provision, users are rewarded with KARMA, encouraging more liquidity in the Chameleon pools.
```
Comment (WRRicht3r) [inline src/karma.md]:
```suggestion
By participating in liquidity provision for this pool, users are rewarded with KARMA, encouraging more liquidity in the Chameleon pools.
```

File: src/karma.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+## Minting and Burning Mechanism
+
+For each Kensetsu XOR (KXOR) minted from collateralized deposits, an additional 1% of KXOR is minted specifically to buy back and burn KARMA tokens.
+
+This buy-back-and-burn mechanism helps to regulate the supply of KARMA, supporting its value.
```
Comment (WRRicht3r) [inline src/karma.md]:
```suggestion
This buy-back-and-burn mechanism helps regulate the KARMA supply, supporting its value.
```

File: src/karma.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+## Redistribution and Farming Rewards
+
+80% of the KARMA tokens that are bought back and burned will be reminted and distributed to the Demeter farm.
+
+These reminted KARMA tokens are then used as farming rewards for liquidity providers in the Chameleon pools, initially focusing on the XOR-ETH pool on Polkaswap.
+
+## Deflationary Aspect
+
+KARMA also has a deflationary mechanism. For every 10 million XOR burned, 1 KARMA token is minted.
```
Comment (WRRicht3r) [inline src/karma.md]: Will more Karma be minted after the initial distribution? or is this describing the burn drop mechanism?

File: src/karma.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+After this initial minting, the KARMA token becomes deflationary, with 1% of minted KXOR used to buy back and burn KARMA tokens.
+
+## In summary
```
Comment (WRRicht3r) [inline src/karma.md]: Please start the article with the summary

File: src/kxor.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
+---
+title: Kensetsu XOR | SORA Docs
+head:
+  - - meta
+    - name: description
+      content: Learn about KENSETSU, Pegged to XOR, swap for ETH in the XOR-ETH pool, buy-back mechanizm, buring KEN, burning TBCD
```
Comment (WRRicht3r) [inline src/kxor.md]:
```suggestion
      content: Learn about Kensetsu XOR (KXOR), a robust and flexible financial tool, pegged to the price of XOR that supports the growth and stability of the SORA ecosystem.  
```

File: src/kxor.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+  - - meta
+    - name: keywords
+      content: token burn, Kensetsu, XOR, ETH, KEN, TBCD
+---
+
+# Kensetsu XOR (KXOR)
+
+Kensetsu XOR (KXOR) is a crucial component of the SORA ecosystem, providing a stable and versatile token for users and builders. Here's everything you need to know about KXOR.
+
+## Pegged to XOR
```
Comment (WRRicht3r) [inline src/kxor.md]: you can remove this subheader and add the next sentence to the intro

File: src/kxor.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+This feature adds significant liquidity and flexibility for users, allowing them to easily convert KXOR into other valuable assets.
+
+## Overcollateralization and Buy-Back Mechanism
+
+When KXOR is minted, it is overcollateralized by 3% to maintain stability and support the ecosystem.
+
+The overcollateralized 3% is distributed as follows:
+
+- 1% goes towards buying back and burning KEN, similar to other Kensetsu assets.
+- 1% is allocated for buying back and burning the Chameleon Token, a new token in the ecosystem.
```
Comment (WRRicht3r) [inline src/kxor.md]: is this PUSSY?

File: src/kxor.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+- 1% is used to buy back and burn TBCD, further stabilizing the token’s value
+
+## Integration with PUSSY Tokens
```
Comment (WRRicht3r) [inline src/kxor.md]: Add a hyperlink to the PUSSY page

File: src/pussy.md [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+head:
+  - - meta
+    - name: description
+      content: Learn about PUSSY, reward builders, mint Kensetsu XOR (KXOR)
+  - - meta
+    - name: keywords
+      content: reward, mint, Kensetsu, XOR, KXOR
+---
+
+# PUSSY :cat:
```
Comment (WRRicht3r) [inline src/pussy.md]: A nice cat image would be good

File: src/pussy.md [hunk 12]
```diff
@@ -0,0 +1,53 @@
+---
+title: PUSSY | SORA Docs
+head:
+  - - meta
+    - name: description
+      content: Learn about PUSSY, reward builders, mint Kensetsu XOR (KXOR)
```
Comment (WRRicht3r) [inline src/pussy.md]:
```suggestion
      content: Learn about the PUSSY token, a dedicated cryptocurrency designed to fund and reward builders within our community
```

File: src/pussy.md [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+  - - meta
+    - name: keywords
+      content: reward, mint, Kensetsu, XOR, KXOR
+---
+
+# PUSSY :cat:
+
+## Introduction
+
+The PUSSY token is a dedicated cryptocurrency designed to fund and reward builders within our community. Here’s an overview of its purpose and functionality
```
Comment (WRRicht3r) [inline src/pussy.md]:
```suggestion
The PUSSY token is a dedicated cryptocurrency designed to fund and reward builders within the SORA community. Here’s an overview of its purpose and functionality
```

File: src/pussy.md [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+
+## Purpose of PUSSY Tokens
+
+The PUSSY token is created specifically to support builders working on productive outputs and projects within the SORA ecosystem.
```
Comment (WRRicht3r) [inline src/pussy.md]:
```suggestion
The PUSSY token supports builders working on productive outputs and projects within the SORA ecosystem.
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

File: src/karma.md (hunk 5 above)
Comment (AndreyKortosov) [reply src/karma.md]: No, no more Karma will be minted. The users have the opportunity to mint Karma via burn XOR until June 06.

File: src/kxor.md (hunk 9 above)
Comment (AndreyKortosov) [reply src/kxor.md]: No

File: src/kxor.md [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+This feature adds significant liquidity and flexibility for users, allowing them to easily convert KXOR into other valuable assets.
+
+## Overcollateralization and Buy-Back Mechanism
+
+When KXOR is minted, it is overcollateralized by 3% to maintain stability and support the ecosystem.
+
+The overcollateralized 3% is distributed as follows:
+
+- 1% goes towards buying back and burning KEN, similar to other Kensetsu assets.
+- 1% is allocated for buying back and burning the Chameleon Token, a new token in the ecosystem.
```
Comment (WRRicht3r) [inline src/kxor.md]:
```suggestion
- 1% is allocated for buying back and burning the Chameleon Token, a new token in the ecosystem proposed by the community.
```

File: src/kxor.md [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+- 1% is used to buy back and burn TBCD, further stabilizing the token’s value
+
+## Integration with [PUSSY Tokens](/pussy.md)
+
+Builders can lock their PUSSY tokens in a Kensetsu vault to mint KXOR.
+
+This process ensures that PUSSY tokens are used productively and enhances the overall utility of the tokens within the ecosystem.
```
Comment (WRRicht3r) [inline src/kxor.md]:
```suggestion
This process ensures that tokens are used productively and enhances the overall utility of the tokens within the ecosystem. The asset used currently is the PUSSY token.
```

File: src/kxor.md [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+
+The overcollateralized 3% is distributed as follows:
+
+- 1% goes towards buying back and burning KEN, similar to other Kensetsu assets.
+- 1% is allocated for buying back and burning the Chameleon Token, a new token in the ecosystem.
+- 1% is used to buy back and burn TBCD, further stabilizing the token’s value
+
+## Integration with [PUSSY Tokens](/pussy.md)
+
+Builders can lock their PUSSY tokens in a Kensetsu vault to mint KXOR.
```
Comment (WRRicht3r) [inline src/kxor.md]:
```suggestion
Builders can lock their tokens in a Kensetsu vault to mint KXOR.
```

File: src/pussy.md [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+      content: reward, mint, Kensetsu, XOR, KXOR
+---
+
+# PUSSY
+
+<img src=".gitbook/assets/pussy_image.png" alt="drawing" width="500"/>
+
+## Introduction
+
+The PUSSY token is a dedicated cryptocurrency in the SORA community, aimed at funding and rewarding builders. It plays a crucial role in the ecosystem by ensuring builders are compensated for their efforts and can convert their earnings into a stable and valuable asset.
```
Comment (WRRicht3r) [inline src/pussy.md]:
```suggestion
The PUSSY token is a dedicated meme cryptocurrency in the SORA economy aimed at funding and rewarding builders. Its role in the ecosystem is to ensure builders are compensated for their efforts and can convert their earnings into a stable and valuable asset.
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

**Review by WRRicht3r — APPROVED** [review]

File: src/karma.md (hunk 5 above)
Comment (WRRicht3r) [inline src/karma.md]:
```suggestion
KARMA also has a deflationary mechanism. During the initial mint, which ended on June 6th, for every 10 million XOR burned, 1 KARMA token was minted.
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

**Review by WRRicht3r — APPROVED** [review]

**Pull request merged**